src/tests/ftest/util/general_utils.py

```diff
@@ -304,3 +307,40 @@ def verify_rebuild(pool, log, to_be_rebuilt, object_qty, record_qty, errors=0):
                 "Unexpected {} value: expected={}, detected={}".format(
                     key, expected, detected))
     return messages
+
+class MyHandler(EventHandler):
+    """
+    Interface to handle events generated by Clustershell
+    """
+    def ev_read(self, worker, node, sname, msg):
```

Comment:
(pylint-unused-argument) Unused argument 'sname'
(pylint-unused-argument) Unused argument 'worker'
(pylint-no-self-use) Method could be a function